docs/why_eyeglass.md

```diff
@@ -0,0 +1,24 @@
+# Why eyeglass?
+eyeglass fills a need for `node-sass` users: access to the rich Sass ecosystem using well-tested node.js development tools. The Sass community has numerous plugins, including [compass](http://compass-style.org), [Susy](http://susy.oddbird.net), [Bourbon](http://bourbon.io), and [many more](http://www.sache.in). If eyeglass is successful, many of these awesome frameworks will become available to `node-sass` users, accessible from a simple `npm install`.
```

Comment:
"If eyeglass is successful" oh ye of little faith.

How about, "By standardizing on eyeglass"